onadata/libs/serializers/merged_xform_serializer.py

```diff
@@ -71,13 +71,29 @@ def get_merged_xform_survey(xforms):
         del merged_xform_dict['_xpath']
 
     is_empty = True
+    xform_dicts = [json.loads(xform.json) for xform in xforms]
     for child in merged_xform_dict['children']:
         if child['name'] != 'meta' and is_empty:
             is_empty = False
         # Remove bind attributes from child elements
         if 'bind' in child:
             del child['bind']
 
+        # merge select one and select multiple options
+        if 'children' in child and child['type'] in SELECTS:
+            children = []
+            for xform_dict in xform_dicts:
+                element_name = child['name']
+                element_list = list(
+                    filter(lambda x: x['name'] == element_name,
+                           xform_dict['children']))
+                if element_list:
+                    element = element_list[0]
+                children += element['children']
```

Comment:
For line 92, is it correct to assume that `element` will always be a non-empty dictionary with a `children` key? What would happen in line 92 if `element_list` in line 90 is empty?

Reply:
moving line 92 into the `if` statement might work

Reply:
going to change these lines to
```
if element_list and element_list[0]:
        children += element_list[0]['children']
```